src/gradata/daemon.py

```
@@ -483,6 +720,74 @@ def _cleanup(self) -> None:
+                pass
+
+        def _send():
+            import platform
+            import urllib.request
+            rules_count = 0
+            lessons_count = 0
+            try:
+                with self._brain_lock:
+                    lessons = self._brain._load_lessons()
+                    lessons_count = len(lessons)
+                    rules_count = sum(1 for l in lessons if l.state.name == "RULE")
```

Ruff (0.15.9)

[error] Ambiguous variable name: `l`

(E741)